### DISCUSSION TITLE
On Refreshing Alerts & Reports page UI disappears

### DISCUSSION THREAD

**Discussion by ukm21** [apache/superset#18297] [Q&A / Help]
When I open Alerts & Reports page I'm able to see the page but if I refresh the browser the UI disappears.

### Expected results

UI should be visible on browser refresh

### Actual results

UI disappears on browser refresh

#### Screenshots

![image](https://user-images.githubusercontent.com/15329626/120476904-c3283600-c3c8-11eb-85e5-c870e26be00a.png)

#### How to reproduce the bug

1. Go to 'Settings -> Alerts & Reports'
2. Refresh the browser
3. See the error "The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again."

### Environment

(please complete the following information):

- superset version: `1.1.0`
- python version: `3.8.9`
- node.js version: `12`

### Checklist

Make sure to follow these steps before submitting your issue - thank you!

- [ ] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [x] I have reproduced the issue with at least the latest released version of superset.
- [x] I have checked the issue tracker for the same issue and I haven't found one similar.

**Comment by zhaoyongjie** [comment]
Hi @ukm21 . You should run `superset init` from the terminal when open alert & report function.

**Comment by ukm21** [comment]
hi @zhaoyongjie  I did superset init but even after that same issue

**Comment by anilvpatel21** [comment]
@ukm21 In the local superset_config.py, you must have enabled ALERT_REPORTS in DEFAULT_FEATURE_FLAGS and remove depreciated ENABLE_ALERTS from the config file. Hope it will work.
```
#ENABLE_ALERTS = False
DEFAULT_FEATURE_FLAGS = {
      ALERT_REPORTS: True
}
```
